shell/src/main/java/org/apache/accumulo/shell/commands/PingCommand.java

```diff
@@ -41,17 +42,17 @@ public String description() {
   public int execute(final String fullCommand, final CommandLine cl, final Shell shellState)
       throws Exception {
 
-    List<String> tservers;
+    final List<String> tservers = new ArrayList<>();
 
     final InstanceOperations instanceOps = shellState.getAccumuloClient().instanceOperations();
 
     final boolean paginate = !cl.hasOption(disablePaginationOpt.getOpt());
 
     if (cl.hasOption(tserverOption.getOpt())) {
-      tservers = new ArrayList<>();
       tservers.add(cl.getOptionValue(tserverOption.getOpt()));
     } else {
-      tservers = instanceOps.getTabletServers();
+      instanceOps.getServers(ServerId.Type.TABLET_SERVER)
```

Comment:
Not a change for this PR.  We may want to expand the set of servers this command works with to include scan servers and maybe compactors.  It could have server type and resource group args.

Reply:
Created #4952